### PR TITLE
Revert to md2man master

### PR DESCRIPTION
**Pull request by baude**
Upstream md2man is working again.  We can revert to using it instead
of a specific commit id.

Also, add make integration.CentOS for testing

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by mheon** [comment]
LGTM

**Comment by mheon** [comment]
Actually, this seems to be missing the revert of the md2man changes @baude 

The integration changes look fine though

File: Makefile [hunk 1]
```diff
@@ -195,7 +199,6 @@ install.tools: .install.gitvalidation .install.gometalinter .install.md2man
 	if [ ! -x "$(GOPATH)/bin/go-md2man" ]; then \
 		   go get -d github.com/cpuguy83/go-md2man; \
 		   cd $(GOPATH)/src/github.com/cpuguy83/go-md2man; \
-		   git checkout 20f5889cbdc3c73dbd2862796665e7c465ade7d1; \
 		   go install github.com/cpuguy83/go-md2man; \
 	fi
```
Comment (TomSweeneyRedHat) [inline Makefile]: Prior this use to be:
go get -u github.com/cpuguy83/go-md2man; \

Are these three lines better than just the one?

Comment (TomSweeneyRedHat) [reply Makefile]: I'm asking not only for here, but I changed it back to just the single line in Buildah and want to adjust if the 3 are better.

Comment (mheon) [reply Makefile]: I think just `go get -u` without the cd and `go install` is sufficient to get it installed, so I think we should remove the other two lines

File: Makefile [hunk 2]
```diff
@@ -193,9 +197,7 @@ install.tools: .install.gitvalidation .install.gometalinter .install.md2man
 
 .install.md2man: .gopathok
 	if [ ! -x "$(GOPATH)/bin/go-md2man" ]; then \
-		   go get -d github.com/cpuguy83/go-md2man; \
-		   cd $(GOPATH)/src/github.com/cpuguy83/go-md2man; \
-		   git checkout 20f5889cbdc3c73dbd2862796665e7c465ade7d1; \
+		   go get -u github.com/cpuguy83/go-md2man; \
 		   go install github.com/cpuguy83/go-md2man; \
```
Comment (mheon) [inline Makefile]: Can remove this too

Comment (baude) [reply Makefile]: does -u imply install

Comment (mheon) [reply Makefile]: Yeah. -u will install if not present, update if present

**Comment by rhatdan** [comment]
bot, retest this please

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**